node_modules/col-canvas/lib/poller.js

```diff
@@ -680,7 +680,11 @@ var handleAssignmentSubmissions = function(ctx, users, assignment, callback) {
         return callback(err);
       }
 
-      log.info({'course': ctx.course.id, 'assignment': assignment.id}, 'Got ' + _.size(submissions) + ' submissions from Canvas');
+      var activeSubmissions = _.reject(submissions, {'workflow_state': 'unsubmitted'});
```

Comment:
@paulkerschen for personal education, I checked [Canvas API doc on workflow_state](https://canvas.instructure.com/doc/api/submissions.html). Are we right to treat 'pending_review' as active? For the record, I have no idea what use case 'pending' represents.